lib/twine/formatters/abstract.rb

```diff
@@ -3,17 +3,28 @@
 module Twine
   module Formatters
     class Abstract
-      attr_reader :strings
-      attr_reader :options
+      attr_accessor :strings
+      attr_accessor :options
 
-      def self.can_handle_directory?(path)
-        return false
-      end
-
-      def initialize(strings, options)
+      def initialize(strings = StringsFile.new, options = {})
```

Comment:
Yeah, if unit tests are the only ones using the parameters at all, we should just remove them.